### PR TITLE
LPS-88713 Add test for creating blog with image

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java [hunk 1]
```diff
@@ -166,6 +168,87 @@ public void testCreateBlog() throws Exception {
 		);
 	}
 
+	@Test
+	public void testCreateBlogPostWithImage() throws Exception {
+		String documentsHref = ApioClientBuilder.given(
```
Comment (cgoncas) [inline modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java]: What do you think about move the creation of the documents to a Test Util? I Think Rubén is doing exactñy the same

File: modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java [hunk 2]
```diff
@@ -331,6 +414,24 @@ private String _read(String fileName) throws Exception {
 		return StringUtil.read(url.openStream());
 	}
 
+	private File _readFile(String fileName) {
```
Comment (cgoncas) [inline modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java]: I think we have this code is more than 3 places, should we move it to a utility class?

**Pull request closed without merging**